images/hook/entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/dumb-init /bin/sh
```

Comment:
Agree, but I think it should be only the entrypoint/cmd, and not this file's shebang interpreter.